scripts/run.sh

```diff
@@ -1,23 +1,26 @@
 #!/bin/bash
+if [ "$FLOW_NETWORK" = "testnet" || "$FLOW_NETWORK" = "mainnet" || "$FLOW_NETWORK" = "canary" || ]; then
```

Comment:
@nialexsan don't we need crescendo in this?

Reply:
@m-Peter said that Crescendo doesn't have evm yet

Reply:
Ah, it's there now but not yet shared on discord since they want to test some things